### PR TITLE
Add listItemSectionBuilder types

### PR DESCRIPTION
**Pull request by giladsegal**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by simast** [comment]
Let's update `src/index.d.ts` as well (since we do now promote named exports):

https://github.com/wix/wix-style-react/blob/1d2d7d5f3422c08a92476e1d9e7072eecf18da9e/src/index.d.ts#L123-L124

File: src/ListItemSection/index.d.ts [hunk 1]
```diff
@@ -6,15 +6,36 @@ export type ListItemSectionTypes =
   | 'title'
   | 'subheader';
 
+export const TYPES: {
```
Comment (mosheKerbel) [inline src/ListItemSection/index.d.ts]: I think we should align with other components
We do not export SKINS, SIZES, etc'...

Comment (giladsegal) [reply src/ListItemSection/index.d.ts]: Well, it is exported from the js code :o

**Comment by giladsegal** [comment]
> Let's update `src/index.d.ts` as well (since we do now promote named exports):
> 
> https://github.com/wix/wix-style-react/blob/1d2d7d5f3422c08a92476e1d9e7072eecf18da9e/src/index.d.ts#L123

Done!

**Pull request merged**